site/src/Librecores/ProjectRepoBundle/Util/DefaultExecutor.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Librecores\ProjectRepoBundle\Util;
+
+use Psr\Log\LoggerInterface;
+
+use Symfony\Component\Process\ProcessBuilder;
+
+/**
+ * Default implementation of ExecutorInterface
+ *
+ * Uses Symfony ProcessBuilder component to build the process.
+ *
+ * @author Amitosh Swain Mahapatra <amitosh.swain@gmail.com>
+ */
+class DefaultExecutor implements ExecutorInterface
```

Comment:
What other ExecutorInterface instances do you expect in the future? Right now, this class hides code and hence makes it less explicit and readable. For example, 
`$builder->setWorkingDirectory(dir)` is much clearer than `exec(..., ['cwd' => dir], ...)`. At the same time, the IDE is able to help finding the right method.